.github/workflows/ci-scheduled-podman-mac.yml

```diff
@@ -0,0 +1,93 @@
+name: Podman Mac CI
+
+on:
+  schedule:
+    - cron: '0 6 * * * ' # Every day at 6:00AM UTC (~midnight CST pending DST)
```

Comment:
Due to the slowness of running MacOS Podman in the Guthub Runner  we opt to run it once a day on some simple tests as a sanity check.